services/storage/tests/unit/test_data_export.py

```diff
@@ -0,0 +1,601 @@
+    _celery_client = _MockCeleryClient(
+        send_task_object=params.get("send_task_object", None),
+        get_task_status_object=params.get("get_task_status_object", None),
+        get_task_result_object=params.get("get_task_result_object", None),
+        get_task_uuids_object=params.get("get_task_uuids_object", None),
+        abort_task_object=params.get("abort_task_object", None),
+    )
+    mocker.patch(
+        "simcore_service_storage.api.rpc._async_jobs.get_celery_client",
+        return_value=_celery_client,
+    )
+    mocker.patch(
+        "simcore_service_storage.api.rpc._data_export.get_celery_client",
+        return_value=_celery_client,
+    )
+    return _celery_client
+
+
+@pytest.fixture
+async def app_environment(
+    app_environment: EnvVarsDict,
+    rabbit_service: RabbitSettings,
+    monkeypatch: pytest.MonkeyPatch,
+):
+    new_envs = setenvs_from_dict(
+        monkeypatch,
+        {
+            **app_environment,
+            "RABBIT_HOST": rabbit_service.RABBIT_HOST,
+            "RABBIT_PORT": f"{rabbit_service.RABBIT_PORT}",
+            "RABBIT_USER": rabbit_service.RABBIT_USER,
+            "RABBIT_SECURE": f"{rabbit_service.RABBIT_SECURE}",
+            "RABBIT_PASSWORD": rabbit_service.RABBIT_PASSWORD.get_secret_value(),
+        },
+    )
+
+    settings = ApplicationSettings.create_from_envs()
+    assert settings.STORAGE_RABBITMQ
+
+    return new_envs
+
+
+@pytest.fixture
+async def rpc_client(
+    initialized_app: FastAPI,
+    rabbitmq_rpc_client: Callable[[str], Awaitable[RabbitMQRPCClient]],
+) -> RabbitMQRPCClient:
+    return await rabbitmq_rpc_client("client")
+
+
+class UserWithFile(NamedTuple):
+    user: UserID
+    file: Path
+
+
+@pytest.mark.parametrize(
+    "location_id",
+    [SimcoreS3DataManager.get_location_id()],
+    ids=[SimcoreS3DataManager.get_location_name()],
+    indirect=True,
+)
+@pytest.mark.parametrize(
+    "project_params,_type",
+    [
+        (
+            ProjectWithFilesParams(
+                num_nodes=1,
+                allowed_file_sizes=(TypeAdapter(ByteSize).validate_python("1b"),),
+                workspace_files_count=10,
+            ),
+            "file",
+        ),
+        (
+            ProjectWithFilesParams(
+                num_nodes=1,
+                allowed_file_sizes=(TypeAdapter(ByteSize).validate_python("1b"),),
+                workspace_files_count=10,
+            ),
+            "folder",
+        ),
+    ],
+    ids=str,
+)
+@pytest.mark.parametrize(
+    "mock_celery_client",
+    [
+        {"send_task_object": TaskUUID(_faker.uuid4())},
+    ],
+    indirect=True,
+)
+async def test_start_data_export_success(
+    rpc_client: RabbitMQRPCClient,
+    mock_celery_client: _MockCeleryClient,
+    with_random_project_with_files: tuple[
+        dict[str, Any],
+        dict[NodeID, dict[SimcoreS3FileID, FileIDDict]],
+    ],
+    user_id: UserID,
+    _type: Literal["file", "folder"],
```

Comment:
please remove the undescore, it's confising since I thought it was something special and call it `selection_type` or soemthing